chart/Chart.yaml

```diff
@@ -19,7 +19,7 @@
 ---
 apiVersion: v2
 name: airflow
-version: 1.5.0-dev
+version: 1.5.1-dev
```

Comment:
I tihnk this should not be changed. We are voting on 1.5.0 and my bet is that we will issue and rc2

Reply:
Yeah, we don't need to touch this here.